app/HMS/Repositories/Banking/Doctrine/DoctrineBankTransactionRepository.php

```diff
@@ -63,6 +63,19 @@ public function findLatestTransactionForAllAccounts()
         return $q->getQuery()->getResult();
     }
 
+    /**
+     * Find the latest transaction.
+     *
+     * @return array
+     */
+    public function findLatestTransaction()
+    {
+        $q = parent::createQueryBuilder('bankTransaction')
+             ->addSelect('MAX(bankTransaction.transactionDate) AS latestTransactionDate');
+
+        return $q->getQuery()->getResult();
```

Comment:
`->setMaxResults(1)` and `getOneOrNullResult()` should return just one BankTransaction

kinda like `findLatestTransactionByAccountGTeAmount`

then[MembershipAuditJob.php:61 won't need the `[0]`